sr/tools/inventory/inventory.py

```diff
@@ -198,15 +219,29 @@ def __init__(self, path, parent=None):
                 self.types[i.name] = []
             self.types[i.name].append(i)
 
+    def _should_ignore(self, fname):
+        """
+        Ignore dotfiles etc.
+        """
+        if should_ignore(fname):
+            return True
+
+        if fname in self.ignore_fnames:
```

Comment:
I might be tempted to make this `return fname in self.ignore_fnames` but I don't mind.

Reply:
Ah, yeah. I originally had some other checks in here, hence the extra step. Going to leave as-is for now.